python/pyspark/sql/pandas/group_ops.py

```diff
@@ -84,6 +94,45 @@ def apply(self, udf):
         jdf = self._jgd.flatMapGroupsInPandas(udf_column._jc.expr())
         return DataFrame(jdf, self.sql_ctx)
 
+    @since(3.0)
+    def applyInPandas(self, func, schema):
+        """
+        It is an alias :meth:`pyspark.sql.GroupedData.apply`; however, it takes a Python
+        native function directly.
```

Comment:
Will `pyspark.sql.GroupedData.apply` be deprecated future? If so, and as it takes Python native func, will it be better not to say it is an alias of `apply`?

Reply:
I will do that when we actually deprecate. If users don't like this way, maybe we should remove this note and don't deprecate.